README.md

```diff
@@ -240,7 +240,7 @@ spec:
     }
 ```
 
-> Note that when returning multiple resources, we need to set different `metadata.name` or `matadata.annotations."krm.kcl.dev/composition-resource-name" ` to distinguish between different resources in the composition functions.
+> Note that when returning multiple resources, we need to set different `metadata.name` or `metadata.annotations."krm.kcl.dev/composition-resource-name" ` to distinguish between different resources in the composition functions.
```

Comment:
Is this really true?

At the Crossplane end, what matters is that the `composition-resource-name` is unique. It's fine for two different composed resources to have the same `metadata.name`, as long as they're of different types.